### ISSUE TITLE
CHANGE BP + ZINC SYRINGE META

### ISSUE THREAD

**Issue by ghost** [HippieStation/HippieStationdeprecated2020#12739]
(very unfun to contend with, especially against tryhard validhunters)

Simple, make bp+zinc syringes (10u diethyl zinc +5u black powder shot out of a syringe gun) not the go to kill antag meta. Couple ideas:

1. Make black powder process through a body faster
2. Make it take more than one 15u syringe to trigger an explosion
3. (your idea here)

**Comment by MrEpicSax** [comment]
I think it should just do less damage when exploding from inside the body and just blow limbs off, I think that would be funnier

**Comment by mkikt4743** [comment]
yeah 1-hit kills really shouldn't be available to crew, also obligatory t. man who died to BP zinc

**Comment by carlopault** [comment]
+2 sure make it take more than 15u. Then the person would have to shoot 2 syringes at their opponent to kill them, which gives the opponent an extra chance to dodge

**Comment by byandaccount** [comment]
+1 one hit kills are fucking lame, the only way to save yourself is by swallowing the ingredients for smoke.

**Comment by AnthonyJNgo** [comment]
+3 for obvious reasons, we need to control meta.

**Comment by Zyzarda** [comment]
+2
